src/steps/user/index.test.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  createMockStepExecutionContext,
+  Recording,
+} from '@jupiterone/integration-sdk-testing';
+
+import { setupSigSciRecording } from '../../../test/recording';
+
+import { IntegrationConfig } from '../../config';
+import { fetchUsers } from './';
+import { integrationConfig } from '../../../test/config';
+import { createOrganizationEntity } from '../organization/converter';
+import { SignalSciencesCorp } from '../../types';
+
+describe('should collect user data', () => {
+  let recording: Recording;
+
+  beforeEach(() => {
+    recording = setupSigSciRecording({
+      directory: __dirname,
+      name: 'buildCorpAndUserRelationship',
+    });
+  });
+
+  afterEach(async () => {
+    await recording.stop();
+  });
+
+  test('validate client integration and converters', async () => {
+    // Arrange
+    const context = createMockStepExecutionContext<IntegrationConfig>({
+      instanceConfig: integrationConfig,
+    });
```

Comment:
Another tip - instead of calling `context.jobState.addEntity()` below, you can actually pass the organizationEntity _directly_ to `createMockStepExecutionContext`.

```suggestion
    const context = createMockStepExecutionContext<IntegrationConfig>({
      instanceConfig: integrationConfig,
      entities: [createOrganizationEntity({ name: 'jupiterone', displayName: 'JupiterOne'})],
    });
```

Then, the organization entity won't be part of  `context.jobState.collectedEntities`, and similar to `organization/index.test.ts', you don't need to `filter()` the `context.jobState.collectedEntities`.

```ts
const users = context.jobState.collectedEntities;
expect(users.length).toBeGreaterThan(0);
expect(users).toMatchGraphObjectSchema({
  _class: Entities.USER._class,
```